src/main/drivers/pwm_output.c

```diff
@@ -779,4 +773,15 @@ void beeperPwmInit(const ioTag_t tag, uint16_t frequency)
     }
 }
 #endif // USE_BEEPER
+
+bool isMotorProtocolDshot(void)
```

Comment:
This can probably move back to where it used to be, and there is no need for the `USE_DSHOT` conditional, as `isDshot` will always be false if `USE_DSHOT` isn't defined.